File: src/main/java/com/eager/questioncloud/authentication/vo/AuthenticationToken.java
package com.eager.questioncloud.authentication.vo;

import lombok.AllArgsConstructor;
import lombok.Getter;

@Getter
@AllArgsConstructor
public class AuthenticationToken {
    private String accessToken;
    private String refreshToken;

    public static AuthenticationToken create(String accessToken, String refreshToken) {
        return new AuthenticationToken(accessToken, refreshToken);
    }
}

File: qc-core/qc-user/qc-user-api/src/main/kotlin/com/eager/questioncloud/user/authentication/model/AuthenticationToken.kt
package com.eager.questioncloud.user.authentication.model

class AuthenticationToken(
    val accessToken: String,
    val refreshToken: String
) {
    companion object {
        fun create(accessToken: String, refreshToken: String): AuthenticationToken {
            return AuthenticationToken(accessToken, refreshToken)
        }
    }
}
